OutOfSchool/OutOfSchool.WebApi/Services/ProviderService.cs

```diff
@@ -178,6 +181,96 @@ public async Task<ProviderDto> Update(ProviderDto providerDto, string userId)
         public async Task<Guid> GetProviderIdForWorkshopById(Guid workshopId) =>
             await workshopServiceCombiner.GetWorkshopProviderId(workshopId).ConfigureAwait(false);
 
+        public async Task<ProviderStatusDto> UpdateStatus(ProviderStatusDto dto, string userId)
```

Comment:
not a call to action, but userId is used only for logging purposes? If it's going to be used later to verify if the given admin can or can not update this status - maybe add a TODO so we won't forget :)